src/components/ProgressBar/index.js

```diff
@@ -0,0 +1,38 @@
+import { createElement as E } from 'react'
+import T from 'prop-types'
+
+import { classNames } from '../../utils/'
+import { NAMESPACE, BLOCK_TAGS } from '../../constants'
+
+const ProgressBar = ({
+  tag,
+  percent,
+  className,
+  children,
+  ...rest }) =>
+  E(
+    tag || 'div',
+    {
+      className: classNames(`${NAMESPACE}c-progress-bar`, className),
+      ...rest
+    },
+    E(
+      tag || 'div',
+      {
+        style: { width: `${percent}%` },
+        className: classNames(`${NAMESPACE}c-progress-bar__indicator`, className),
+        ...rest
+      },
+      children
+    ),
+  )
+
+
+ProgressBar.propTypes = {
+  tag: T.oneOf(BLOCK_TAGS),
+  className: T.string,
+  percent: T.string,
+  children: T.node.isRequired
```

Comment:
When we split out the components you won't want any children prop on `<ProgressBarIndicator />`

Reply:
done